fv/bpf_test.go

```diff
@@ -1351,6 +1351,12 @@ func describeBPFTests(opts ...bpfTestOpt) bool {
 									// Add a route to felix[1] to be able to reach the nodeport
 									_, err = eth20.RunCmd("ip", "route", "add", felixes[1].IP+"/32", "via", "10.0.0.20")
 									Expect(err).NotTo(HaveOccurred())
+									// This multi-NIC scenario works only if the kernel's RPF check
```

Comment:
Can you explain to me what goes wrong here?  Can the test be adjusted to set up working routing instead?

Reply:
When we receive a NP packet on an iface the encaped packet pkt will have src IP == node IP which is local but from a different iface - the one through which it will be most likely forwarded. Seems like strict rpf does not like it even when accept local is enabled.

I think setting up local routes would fix it. It would be the same as the dual ToR (?) in GCE.

Reply:
For the sake of this one test, imo it does not matter how we fix the problem as in production, things will highly depend on how the networking is configured. So I am thinking about leaving it this way for now.